docs/source/isaura.rst

```diff
@@ -1,5 +1,163 @@
 Isaura
-==========
+======
 
-.. caution::
-  This page is currently not created, we are working on it. If you would like to contribute on this documentation, reach `me <helena.almamol@gmail.com>`_!
+*From ancient greek, Ισαυρία: an ancient rugged region in Asia Minor.*
+
+This city computes tracks and extracts topology information from **deconvoluted hits**. Therefore, it is analogous to the final stage of :doc:`esmeralda`. Unlike the latter, the input hits correspond to the deconvoluted ones coming from :doc:`beersheba` (instead of :doc:`penthesilea`), allowing us to obtain a finer tracking of events.
+
+.. _Isaura input:
+
+Input
+-----
+
+ * ``/Run/events``
+ * ``/Run/runInfo``
+ * ``/DECO/Events``
+ * ``/DST/Events``
+
+.. _Isaura output:
+
+Output
+------
+
+ * ``/Tracking/Tracks``: tracking-related information of events (in particular: track energy (``energy``), track length (``length``), number of voxels (``numb_of_voxels``), number of hits (``numb_of_hits``), minimum and maximum position computed with the hits comprising the track (``x_min``, ``y_max``, ...), blobs position (``blob1_z``, ``blob2_x``,...) and energy (``eblob1``, ``eblob2``), energy of the hits shared by both blobs (``ovlp_blob_energy``), and voxel size (``vox_size_x``, ``vox_size_y``, ``vox_size_z``). Each row corresponds to a different track, specified among the others within an event with its ``trackID``.
+ * ``/Summary/Events``: global information related to the event. Each row is one event.
+ * ``/DST/Events``: copy of the point-like information (*kdst*) of events, output of :doc:`penthesilea`.
+ * ``/Filters/hits_select``: flag to indicate if an event did not pass due to the lack of hits.
+ * ``/Filters/topology_select``: flag that indicates whether events have too many hits according to the value specified in the config file. If so, its ID will not appear in the table and the topology information will not be computed.
+ * MC info: copy of the Monte Carlo information for the events that output the city. Only if run number < 0. The tables included are: ``/MC/configuration``, ``/MC/hits``, ``/MC/particles``, ``/MC/sns_positions``, and ``/MC/sns_response``.
```

Comment:
`copy of the Monte Carlo information for the events that output the city`: what do you mean with "events that output the city"? Events that appear in the output of the city? In that case, I think it should be "for the events that pass the filters of the city" or "for the events that the city outputs", if you use `to output` as a synonymous of `to generate`.